partners/scientific-communities.md

```diff
@@ -1,51 +1,64 @@
+support [Pangeo](https://pangeo.io/).
 
 If you are submitting a package for review that fits into the Pangeo ecosystem,
 your package will be:
 
-* Reviewed against current pyOpenSci guidelines and checks. 
+* Reviewed against current pyOpenSci guidelines and checks.
 * Also reviewed against the Pangeo-specific guidelines below.
--->
 
-<!-- 
-```{admonition} Pangeo Review Guidelines
+
+
+```{admonition} Pangeo Specific Software Peer Review Guidelines
 :class: note
 
-On top of the pyOpenSci review guidelines, we will also perform checks for 
-the following: 
+In addition to [basic pyOpenSci review checks](https://www.pyopensci.org/software-peer-review/how-to/editor-in-chief-guide.html#editor-checklist-template), when reviewing a
+Pangeo-related package, we will also perform checks for the following:
+
+* **Consume and produce high-level data structures** (e.g. **Xarray datasets**, **Pandas DataFrames**) wherever feasible:
+* **Operate lazily when called on Dask data structure:** Lazy loading refers to a tool only performing operations on data when it's needed. Lazy loading is ideal for workflows using larger datasets which will quickly consume memory and compute power when you work with them. Lazy loading allows a tool to create a connection to a dataset, but only actually access / open the part of the dataset that you require to use in your workflow.  [Learn more here.](https://docs.dask.org/en/stable/user-interfaces.html?highlight=lazy#laziness-and-computing)
 
-* **Consume and produce high-level data structures** (e.g. **Xarray datasets**, **Pandas DataFrames**) wherever feasible: 
-* **Operate lazily when called on Dask data structure:** Lazy loading refers to a tool only performing operations on data when it's needed. Lazy loading is ideal for workflows using larger datasets which will quickly consume memory and compute power when you work with them. [Learn more here.](https://saturncloud.io/blog/a-data-scientist-s-guide-to-lazy-evaluation-with-dask/)  
-TODO: Is this the best link for an overview of lazy loading - maybe there is a tutorial?? 
-* **Avoid file I/O unless your package is specifically an I/O package:** I/0 refers to data input and output of data.
-TODO: Ask tom for a better explanation of what this specific means and add a link if there is one ? 
-``` -->
+    * A few other resources related to lazy loading:
+
+        * [Dask and stages of computation](https://docs.dask.org/en/stable/phases-of-computation.html): this provides and overview of using Dask effectively to optimize your compute workflow.
+        * [Using Dask and Xarray together:](https://docs.xarray.dev/en/stable/user-guide/dask.html?highlight=lazy#using-dask-with-xarray) this page provides an overview of how to use Dask to optimize working with data arrays in Xarray.
+
+* **Avoid file input and output (I/O) unless your package is specifically an I/O package:** I/0 refers to data input and output of data.
```

Comment:
i need some clarification on what this means in practice as it's often useful to write out a file during a workflow as an artifact or for use elsewhere. even xarray i think allows this and also needs to open datasets?

Reply:
maybe this relates to https://docs.2i2c.org/en/latest/data/cloud.html#cloud-native-formats so rather than simple exporting say txt you want a cloud native format?

Reply:
Use existing data reading and writing libraries, rather than writing your own. Tools like xarray and it's readers and writers, which build on libraries like Zarr, fsspec, h5netcdf, GDAL, and more can already read and write a variety of file types to a variety of file systems, including Cloud Object storage.

Reply:
i grabbed a bit of language from your pangeo guide here too

is this the text below ok @TomAugspurger  ?
***************
* **Use existing data reading and writing libraries rather than creating your own read and write utilities:** Unless you package is specifically focused on reading or writing data with specific use cases in mind, it
   should not include its own custom code for read/write data operations. The data be read and written using tools within packages such as Xarray or geoparquet via GeoPandas that build on existing packages such as Zarr, fsspec, h5netcdf, and GDAL. These tools already read and write a variety of file types to a variety of file systems, including Cloud Object storage.
[See: 2i2c documentation for more on cloud native file format support.](https://docs.2i2c.org/en/latest/data/cloud.html#cloud-native-formats)